core/src/main/java/org/mskcc/cbio/oncokb/util/EvidenceTypeUtils.java

```diff
@@ -0,0 +1,66 @@
+package org.mskcc.cbio.oncokb.util;
+
+import org.mskcc.cbio.oncokb.model.EvidenceType;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Created by Hongxin on 11/09/16.
+ */
+public class EvidenceTypeUtils {
+    public static Set<EvidenceType> getGeneEvidenceTypes() {
+        Set<EvidenceType> evidenceTypes = new HashSet<>();
+        evidenceTypes.add(EvidenceType.GENE_SUMMARY);
+        evidenceTypes.add(EvidenceType.GENE_BACKGROUND);
+        return evidenceTypes;
+    }
+
+    public static Set<EvidenceType> getMutationEvidenceTypes() {
+        Set<EvidenceType> evidenceTypes = new HashSet<>();
+        evidenceTypes.add(EvidenceType.ONCOGENIC);
+        evidenceTypes.add(EvidenceType.MUTATION_EFFECT);
+        evidenceTypes.add(EvidenceType.VUS);
+        evidenceTypes.add(EvidenceType.MUTATION_SUMMARY);
```

Comment:
We don't use MUTATION_SUMMARY any more right? The mutation summary is contained in the MUTATION_EFFECT type evidence description.

Reply:
We haven't remove MUTATION_SUMMARY from the system yet. Will do in the same batch.